### ISSUE TITLE
[Spree 2.1] Cannot connect Stripe

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#5437]
## Description

I've tried to connect my stripe test account with my enterprise. I got an authorization failure.

## Steps to Reproduce
1. Login with rachel.arnould+spree[at]protonmail.com and my infamous password
2. Go to payment method and try to connect a Stripe account

## Animated Gif/Screenshot

![image](https://user-images.githubusercontent.com/6525576/82030755-24377b80-9699-11ea-86c0-64e09c349294.png)

## Severity
bug-s1: a critical feature is broken: checkout, payments, signup, login

## Your Environment
* Version used: v.3
* Browser name and version: Firefox 76
* Operating System and version (desktop or mobile): Desktop

**Comment by luisramos0** [comment]
I am not sure exactly what went wrong for you but the staging server was connected to the UK LIVE account (not the test mode). I cant switch to the test mode without the credentials, someone with access to uk stripe account will have to do it: I created an issue to update ofn-secrets with those UK test credentials: https://github.com/openfoodfoundation/ofn-secrets/issues/37

Meanwhile I switched uk staging to use my stripe account and it all works fine for me.

Can you test now Rachel?

**Comment by luisramos0** [comment]
ping @Matt-Yorkley I updated stripe details in uk staging.

**Comment by filipefurtad0** [comment]
Hi @luisramos0,
I tried connecting my stripe account and it worked, as well as the basic functionality:
- Stripe Connect payments, with the 42424... card.
- Stripe SCA payments, with the 42424... card.
- Stripe SCA fails when using a 3D secure card: authentication page leads to broken links - this is now issue #5447 

I observed an error 404 when creating a Stripe payment method. This happened by leaving the stripe owner field empty, I reproduced this a couple of times. Then all of the sudden, it started working as it should, and displaying the expected warning:

![image](https://user-images.githubusercontent.com/49817236/82162141-2ae11100-989a-11ea-973f-1c8f78af168e.png)

I did not open an issue on this, as it now works.
